src/pudl/metadata/resources/ferc714.py

```diff
@@ -61,10 +74,26 @@
     },
     "out_ferc714__hourly_planning_area_demand": {
         "description": (
-            "This table is based on ``core_ferc714__hourly_planning_area_demand``, but adds "
-            "imputed demand values where the original data was missing or anomalous. "
-            "Codes explaining why values have been imputed can be found in the "
-            "``core_pudl__codes_imputation_reasons`` table."
+            {
+                "additional_summary_text": "electricity demand by planning area.",
+                "additional_source_text": "(Part III, Schedule 2a)",
+                "usage_warnings": [
+                    "imputed_values",
+                    {
+                        "type": "custom",
+                        "description": (
+                            "The datetime_utc timestamps have been cleaned due to "
+                            "inconsistent datetime reporting. See below for additional details."
+                        ),
```

Comment:
details not actually provided below! maybe link to :ref:core_ferc714__hourly_planning_area_demand instead?

```suggestion
                            "inconsistent datetime reporting. See :ref:`core_ferc714__hourly_planning_area_demand` for additional details."
                        ),
```